.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
```

Comment:
Looks like the versions in here could get a bump. You can copy this one:

https://github.com/trussworks/terraform-aws-logs/blob/master/.pre-commit-config.yaml

Reply:
Done.